trigger/twister3.py

```diff
@@ -217,7 +224,15 @@ def is_awaiting_confirmation(d):
                 log.msg('[%s] Waiting %s seconds before sending next command' %
                         (self.hostname, self.command_interval))
 
-            reactor.callLater(self.command_interval, self._send_next)
+                reactor.callLater(self.command_interval, self._check_results)
+
+    def _check_results(self):
+        payload = list(reversed(self.results))[:len(self.commands)]
+        log.msg("PAYLOAD: {}".format(payload))
+        payload.reverse()
```

Comment:
This is confusing just like it was before! Why are you reversing the payload and then re-reversing it? Can you gratuitously comment to explain?